### ISSUE TITLE
Section 3

### ISSUE THREAD

**Issue by JSainath** [trevorlolsen/outline#14]
<img width="688" alt="Screen Shot 2020-06-18 at 6 32 07 PM" src="https://user-images.githubusercontent.com/65315822/85084267-8d774680-b191-11ea-9d9a-12bd7024c444.png">

<img width="553" alt="Screen Shot 2020-06-18 at 6 32 15 PM" src="https://user-images.githubusercontent.com/65315822/85084278-90723700-b191-11ea-85f3-1b29a5cc07fc.png">

**Comment by trevorlolsen** [comment]
Thanks, I'll work on these changes.

**Comment by JSainath** [comment]
<img width="636" alt="Screen Shot 2020-06-18 at 9 26 45 PM" src="https://user-images.githubusercontent.com/65315822/85093582-ece15080-b1a9-11ea-993e-be8b39b3cb5e.png">

**Comment by trevorlolsen** [comment]
No, the method works with any number of components. If there are L mixture components, then each individuals betas still only follows one of the L normal distributions per draws. They are not being weighted from all L at the same time (at least that is how I understand RAM version of the model). So, each beta is still distributed normally and section 3.2.2 should apply (I believe).

**Comment by trevorlolsen** [comment]
Also, I'm about to push some changes to section 3 based on the comments from Bill.

**Comment by JSainath** [comment]
> maybe, but the method works with any number of components. If there are L mixture components, then each individuals betas still only follows one of the L normal distributions per draw. They are not being weighted from all L at the same time (at least that is how I understand RAM version of the model). So, each beta is still distributed normally if they never switch latent segments.
> 
> If there is a lot of latent class switching for respondent n and their parameters are not just normally distribution, then the method still works, but the language might be wrong like you are pointing out. Train shows that the f :
> ![image](https://user-images.githubusercontent.com/47762165/85095314-3d5aad00-b1ae-11ea-88f4-45ef3cacd98c.png)
> 
> can be anything you want. But calling it the log normal distribution may not make sense like you are saying if the underlining beta follows a mixture distribution.

Ok, let me take another look at RAM and get back on that. You can continue working on Bill's suggestions.

**Comment by JSainath** [comment]
> > maybe, but the method works with any number of components. If there are L mixture components, then each individuals betas still only follows one of the L normal distributions per draw. They are not being weighted from all L at the same time (at least that is how I understand RAM version of the model). So, each beta is still distributed normally if they never switch latent segments.
> > If there is a lot of latent class switching for respondent n and their parameters are not just normally distribution, then the method still works, but the language might be wrong like you are pointing out. Train shows that the f :
> > ![image](https://user-images.githubusercontent.com/47762165/85095314-3d5aad00-b1ae-11ea-88f4-45ef3cacd98c.png)
> > can be anything you want. But calling it the log normal distribution may not make sense like you are saying if the underlining beta follows a mixture distribution.
> 
> Ok, let me take another look at RAM and get back on that. You can continue working on Bill's suggestions.

This is fine.

**Comment by trevorlolsen** [comment]
ok, thanks.

**Comment by JSainath** [comment]
<img width="315" alt="Screen Shot 2020-06-20 at 4 31 53 PM" src="https://user-images.githubusercontent.com/65315822/85215124-5e97d680-b331-11ea-80b7-faf2569e9edd.png">

**Comment by trevorlolsen** [comment]
![](https://user-images.githubusercontent.com/65315822/85215124-5e97d680-b331-11ea-80b7-faf2569e9edd.png)


Thanks! Instead of defining partial utility, I just changed the words to portion of utility:

![image](https://user-images.githubusercontent.com/47762165/85215562-c81ae380-b337-11ea-8f77-308dea292f6e.png)


Also,  changed to \implies for the differences coding section

![image](https://user-images.githubusercontent.com/47762165/85215571-de28a400-b337-11ea-94cf-255d00da1fd7.png)


Thanks again.

**Comment by JSainath** [comment]
Ok!

**Comment by JSainath** [comment]
<img width="475" alt="Screen Shot 2020-06-20 at 10 40 10 PM" src="https://user-images.githubusercontent.com/65315822/85216827-ab39dc80-b346-11ea-99b7-652143d8f8ff.png">

**Comment by JSainath** [comment]
<img width="390" alt="Screen Shot 2020-06-20 at 11 39 19 PM" src="https://user-images.githubusercontent.com/65315822/85217466-c27cc800-b34e-11ea-8e20-cf2c23acb199.png">

**Comment by trevorlolsen** [comment]
This is disconcerting.  The current content has shifted, so this specific occurrence is no longer a problem.  Not sure how to prevent in the future.

**Comment by JSainath** [comment]
You might have to fix page margin parameters. The easier fix may be to place the footer at the right edge of the page.
